integration/samples.sh

```diff
@@ -1,37 +1,60 @@
-#!/bin/bash
-set -e
-
-FIND_COMMAND=""
-
-if [[ $OSTYPE == "win32" ]]; then
-    FIND_COMMAND="dir /b /o:n /ad"
```

Comment:
@alan-agius4, can you please check what's failing? I don't have access to the logs. It's probably the removal of this line, but I'm wondering why this is needed. What are the pipelines for Windows are using? Is it `MinGW`?